definitions/ext-dora/definition.yml

```diff
@@ -0,0 +1,35 @@
+domain: EXT
+type: DORA
+synthesis:
+  rules:
+    - identifier: repoName
```

Comment:
Hi,

I just wondered if `repoName` would be unique enough within a customer account. If there are two repos with the same name, all tags/telemetry would end up on one entity mixed.

The tags projectId and repoUrl made me think about it, they look "more unique", but I am not sure if they are guaranteed to be there in any case ?

Reply:
Hi @andre-nr , 

Based on conversations with @naxhh and team, have updated the identifier.